source/guides/1.1/views/basic-usage.md

```diff
@@ -86,6 +86,25 @@ end
 Rendering context for a template is made of view methods and exposures.
 </p>
 
+The objects exposed in the controller action are available in the corresponding view. So the values 
+can also be modified, wrapped or reused in some other way. Assuming that the `title` is exposed
+in the action, it can be accessed as follows:
+
+```ruby
+# apps/web/views/dashboard/index.rb
+module Web::Views::Dashboard
```

Comment:
WDYT maybe it will be good to add controller class with `title` expose?

Reply:
It is defined two blocks of code before (in the same section). I don't think it would be helpful to write it once again.

Reply:
great point, thanks